examples/main.go

```diff
@@ -188,6 +193,13 @@ func (c *Counter) HandleEvent(ctx context.Context, e *live.Event) error {
 			// TODO: nil, nil, etc.
 			c.Changeset = cc.NewChangeset(nil, nil, "", &Person{})
 		}
+	case "redirect":
+		// redirect to the given path
+		url, e := url.Parse("/dead")
+		if e != nil {
+			return e
+		}
+		live.Redirect(ctx, *url)
```

Comment:
Do APIs typically deal in `*url.URL`s or the flat struct? It looks a little weird to dereference it here

Reply:
Definitely `*url.URL`

Reply:
cool